src/app/frontend/uicomponents/uicomponents_module.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import helpSectionDirective from './helpsection/helpsection_directive';
+import userHelpDirective from './helpsection/userhelp_directive';
+
+/**
+ * Angular module for elements reused throughout dashboard.
+ *
+ */
+export default angular.module(
```

Comment:
+1 

I agree. If you ain't gonna need it(YAGNI :)) in any other place right now then it's better to tie it with one module only.